src/app/components/Staking/Staking.tsx

```diff
@@ -136,7 +139,7 @@ export const Staking: React.FC<StakingProps> = ({
 
   // load global params and calculate the current staking params
   const globalParams = useGlobalParams();
-  useMemo(() => {
+  useEffect(() => {
```

Comment:
Are we okay using `useEffect` without the dependencies array? Any future concerns with the multiple re-rendering or performance?

Reply:
@gbarkhatov sorry, I don't follow, this particular `useEffect` has dependency array or do I miss something?

Reply:
Nvm